app/src/main/java/ch/epfl/sweng/swenggolf/offer/DatabaseOfferConsumer.java

```diff
@@ -0,0 +1,10 @@
+package ch.epfl.sweng.swenggolf.offer;
+
+import java.util.List;
+
+import ch.epfl.sweng.swenggolf.database.Database;
+import ch.epfl.sweng.swenggolf.database.ValueListener;
+
```

Comment:
Add Javadoc for the interface and the class

Reply:
I added the Javadoc.